Source/Dafny/DafnyPipeline.csproj

```diff
@@ -1,227 +1,35 @@
-    <CodeContractsUseBaseLine>False</CodeContractsUseBaseLine>
-    <CodeContractsEmitXMLDocs>False</CodeContractsEmitXMLDocs>
-    <CodeContractsCustomRewriterAssembly />
-    <CodeContractsCustomRewriterClass />
-    <CodeContractsLibPaths />
-    <CodeContractsExtraRewriteOptions />
-    <CodeContractsExtraAnalysisOptions />
-    <CodeContractsBaseLineFile />
-    <CodeContractsCacheAnalysisResults>False</CodeContractsCacheAnalysisResults>
-    <CodeContractsRuntimeCheckingLevel>Full</CodeContractsRuntimeCheckingLevel>
-    <CodeContractsReferenceAssembly>Build</CodeContractsReferenceAssembly>
-    <CodeContractsAnalysisWarningLevel>0</CodeContractsAnalysisWarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-<PropertyGroup>
-  <PreBuildEvent>
-    "..\..\..\..\third_party\Coco\bin\Coco.exe" "../../../../Source/Dafny/Dafny.atg" "-namespace" "Microsoft.Dafny" "-frames" "../../../../third_party/Coco/src" 
-  </PreBuildEvent>
-</PropertyGroup>
-  <ItemGroup>
-    <Reference Include="BoogieBasetypes">
-      <HintPath>..\..\..\boogie\Binaries\BoogieBasetypes.dll</HintPath>
-    </Reference>
-    <Reference Include="BoogieCore">
-      <HintPath>..\..\..\boogie\Binaries\BoogieCore.dll</HintPath>
-    </Reference>
-    <Reference Include="BoogieParserHelper">
-      <HintPath>..\..\..\boogie\Binaries\BoogieParserHelper.dll</HintPath>
-    </Reference>
-    <PackageReference Include="Mono.Cecil" Version="0.11.2" />
-    <Reference Include="System" />
-    <Reference Include="System.Core">
-      <RequiredTargetFramework>4.8</RequiredTargetFramework>
-    </Reference>
-    <Reference Include="System.Numerics" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Cloner.cs" />
-    <Compile Include="Compiler-Csharp.cs" />
-    <Compile Include="Compiler-go.cs" />
-    <Compile Include="Compiler-js.cs" />
-    <Compile Include="Compiler-cpp.cs" />
-    <Compile Include="Compiler-java.cs" />
-    <Compile Include="Reporting.cs" />
-    <Compile Include="Triggers\QuantifiersCollection.cs" />
-    <Compile Include="Triggers\QuantifierSplitter.cs" />
-    <Compile Include="Triggers\TriggerExtensions.cs" />
-    <Compile Include="Triggers\QuantifiersCollector.cs" />
-    <Compile Include="Triggers\TriggersCollector.cs" />
-    <Compile Include="Triggers\TriggerUtils.cs" />
-    <Compile Include="Util.cs" />
-    <Compile Include="Compiler.cs" />
-    <Compile Include="BigIntegerParser.cs" />
-    <Compile Include="DafnyAst.cs" />
-    <Compile Include="DafnyMain.cs" />
-    <Compile Include="DafnyOptions.cs" />
-    <Compile Include="Printer.cs" />
-    <Compile Include="RefinementTransformer.cs" />
-    <Compile Include="Resolver.cs" />
-    <Compile Include="Rewriter.cs" />
-    <Compile Include="SccGraph.cs" />
-    <Compile Include="Translator.cs" />
-    <Compile Include="..\version.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="cce.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Dafny.atg" />
```

Comment:
Does this need to be included on the right to trigger recompilation when it's changed?